forte2/orbitals/aset.py

```diff
@@ -149,10 +149,14 @@ def _parse_fragment(self, frag_str: str | list[str]) -> list[int]:
         list[int]
             A sorted list of unique atom indices (0-based) matching the specification.
         """
+        if isinstance(frag_str, str):
+            frag_str = [frag_str]
+        else:
+            frag_list = frag_str
```

Comment:
Is line 153 supposed to assign the list to `frag_list`? If yes, then we should replace these four lines with the one-liner:
```
frag_list = [frag_str] if isinstance(frag_str, str) else frag_str
```